build.alpine.pkr.hcl

```diff
@@ -19,6 +19,15 @@ build {
     ]
     execute_command = "echo 'vagrant' | sudo -S -E sh -c '{{ .Vars }} {{ .Path }}'"
   }
+  
```

Comment:
Why not just use a single file provisioner and point the destination to the `/etc/profile.d` directory?

Reply:
Can't do that for `[root directories.](https://www.packer.io/docs/provisioners/file)` You have to use `inline` to move the files to root. I think I got an error when I tried using `execute_command = "echo 'vagrant' | sudo -S -E sh -c '{{ .Vars }} {{ .Path }}'"`